Tasks/AzureRmWebAppDeployment/AzureRmWebAppDeployment-Legacy.ps1

```diff
@@ -37,7 +37,13 @@ param
 	[String] $AdditionalArguments,
```

Comment:
If we are done with testing with PS3 as handler for this task, we should remove all the legacy code. We should not invest more on it, that we plan to remove sooner.